### DISCUSSION TITLE
How can I reopen the access point to change my configuration?

### DISCUSSION THREAD

**Discussion by Zimbu98** [ESPresense/ESPresense#924] [Q&A] [answered]
As the title says, how do I do this (other than having to reflash the firmware)?

**Comment by Telectroboy** [comment, marked as answer]
If you switch off your wifi a moment. The ESP32 will reopen the Access Point after a about a minute.
You can connect to the ESPresense and change your wifi credentials.

**Comment by Terastar-Paperclip** [comment]
Telectroboy's AP-reopen workaround above works. For changing WiFi credentials without touching the AP, see [#746](https://github.com/ESPresense/ESPresense/discussions/746) — DT's MQTT command updates SSID across all nodes at once. Cross-linking for visibility.
